src/rust/bitbox02-rust/src/workflow/transaction.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::bb02_async::option;
+use core::cell::RefCell;
+
+extern crate alloc;
+use alloc::boxed::Box;
+
+pub struct UserAbort;
+
+pub async fn verify_recipient(recipient: &str, amount: &str) -> Result<(), UserAbort> {
```

Comment:
these don't seem to be used anywhere? did you mean to add another commit to replace bits in C to call these functions?

Reply:
Used in this follow up PR:

https://github.com/digitalbitbox/bitbox02-firmware/pull/720

Reply:
:+1: thanks. good to know :)